exalearn-ui/src/app/auth.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
```

Comment:
As I see, service is not inside 'services' folder. Please, move it if I right

Reply:
Moved to service directory